Ana_Lisa_Sutherland_Octos.rb

```diff
@@ -0,0 +1,92 @@
+class Solar_S
+  attr_accessor :planets
+
+  def initialize (planets)
+    @planets = planets
+  end
+  # puts out list of just the planets in the solar system
+  def summary
+    x = 1
+    planet_sum = []
+    @planets.each do |planet|
+      planet_sum << "#{x}. #{planet.name}"
+      x += 1
+    end
+    return planet_sum
+  end
+  # allows user to pick a planet amongst the solar system
+end
+
+class Planet
+  ## Create reader methods to give a user access to read the instance variables.
+  attr_accessor :name, :color, :order, :year_length, :distance_from_the_sun
```

Comment:
Why make these `attr_accessor`  unless you specifically want to give users the ability to directly change the attributes using `attr_reader` would be more appropriate.